src/Components/test/testassets/BasicTestApp/BasicTestApp.csproj

```diff
@@ -21,6 +21,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\..\Web.Extensions\src\Microsoft.AspNetCore.Components.Web.Extensions.csproj" />
```

Comment:
You should be able to use `<Reference Include="Microsoft.AspNetCore.Components.Web.Extensions" />` once you run https://github.com/dotnet/aspnetcore/blob/master/eng/scripts/GenerateProjectList.ps1